geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseIndexAdapter.scala

```diff
@@ -42,7 +45,15 @@ trait HBaseIndexAdapter extends HBaseFeatureIndexType
     val put = new Put(row)
     feature.values.foreach(v => put.addImmutable(v.cf, v.cq, v.value))
     feature.visibility.foreach(put.setCellVisibility)
-    put
+    SystemProperty("geomesa.hbase.wal.durability").option match {
+      case Some(value) =>
+        val durability = Durability.values.find(_.toString.equalsIgnoreCase(value)).getOrElse{
+          logger.error(s"Invalid HBase WAL Durability setting: ${value}. Falling back to default Durability.")
+          Durability.USE_DEFAULT
+        }
```

Comment:
we shouldn't execute this code for every insert - can we move it to a `lazy val Option[Durability]` in the companion object? we wouldn't be able to update it at runtime then, but that should probably be ok

Reply:
Good point, I didn't realize this was run on every row.